AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/booking-details/booking-details.component.ts

```diff
@@ -223,7 +223,11 @@ export class BookingDetailsComponent implements OnInit, OnDestroy {
         if (status === UpdateBookingStatus.Cancelled) {
```

Comment:
Could you also add a unit test to make sure `this.videoHearingService.getHearingById(this.hearingId)` is never called if the status is failed.

Reply:
it is already there and this is not called, pls refer to test 'should show pop up if the confirm failed' in booking-details.component.spec.ts